docs/source/index.rst

```diff
@@ -178,7 +178,7 @@ Miscellaneous Designs
    :caption: USB Designs
 
    pages/BW1098OAK.rst
-   pages/BK1096.rst
```

Comment:
Could you remove `BK1096.rst` page? It seems unused

Reply:
@VanDavv, thanks for noticing! I will use it again.